Shortly.Codes.Application/Exceptions/CodeAlreadyExists.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Net;
+
+namespace Shortly.Codes.Application.Exceptions;
+
+public class CodeAlreadyExists : HttpRequestException
+{
+    public const string MESSAGE = "error.code.already.exists";
```

Comment:
`type.assembly/module.payload`
```suggestion
    public const string MESSAGE = "error.code.alreadyExists";
```